content/06.discussion.md

```diff
@@ -269,13 +269,42 @@ comparable performance. Towards this end, Che et al. [@tag:Che2015_distill] used
 gradient boosted trees to learn interpretable healthcare features from trained
 deep models.
 
-Finally, it is sometimes possible to train the model to provide justifications
+It is sometimes possible to train the model to provide justifications
 for its predictions. Lei et al. [@tag:Lei2016_rationalizing] used a generator to
 identify "rationales", which are short and coherent pieces of the input text
 that produce similar results to the whole input when passed through an encoder.
 The authors applied their approach to a sentiment analysis task and obtained
 substantially superior results compared to an attention-based method.
 
+Lastly, interpretation of embedded or latent space features learned through
```

Comment:
Can we use one sentence per line for new paragraphs?

Reply:
Sure. I was doing this originally, but the rest of the discussion does not have this format. I'll update in a new commit soon